ics/alarm.py

```diff
@@ -157,6 +157,9 @@ def __repr__(self):
 
         return '<{0}>'.format(value)
 
+    def __hash__(self):
```

Comment:
Added the hash method to the Alarm class to be able to add a list of Alarm to the Event.alarm set